test/utils.js

```diff
@@ -103,8 +106,8 @@ function releaseChart(chart) {
 function triggerMouseEvent(chart, type, el) {
 	var node = chart.canvas;
 	var rect = node.getBoundingClientRect();
-	var x = el ? el.x !== undefined ? el.x : el._model.x : null;
-	var y = el ? el.y !== undefined ? el.y : el._model.y : null;
+	var x = el ? el.x !== undefined ? el.x : el.getProps(['x']).x : null;
```

Comment:
I don't think using `el.getProps` gives any additional success here.
`var x = el && el.x !== undefined ? el.x : null` should work the same.